src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -83,6 +84,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        model.updateFilteredTutorialList(PREDICATE_HIDE_ALL_TUTORIALS);
```

Comment:
Is this done to direct focus of user to person list after editing a person?

*and also subsequently in ListTutorial Command.

Reply:
yes it is!